components/Gallery/index.js

```diff
@@ -1,26 +1,40 @@
 //Import external resources
```

Comment:
I like your way to comment your imports, makes a lot of sence and easier to read.

Reply:
Yeah it makes it more easy to navigate through the imports.
I can also see if I forgot something.
It also helps me to maintain the app better in the future. Just in case one of the external lib doesn't work anymore I can find and replace them more easily.